esp-hal-embassy/src/lib.rs

```diff
@@ -69,24 +69,24 @@ pub trait TimerCollection {
 
 /// Helper trait to reduce boilerplate.
 ///
-/// We can't blanket-implement for `Into<ErasedTimer>` because of possible
+/// We can't blanket-implement for `Into<AnyTimer>` because of possible
 /// conflicting implementations.
-trait IntoErasedTimer: Into<ErasedTimer> {}
+trait IntoErasedTimer: Into<AnyTimer> {}
```

Comment:
I suppose we should rename this `Into` trait too, just for consistency

Reply:
fair